lib/trailblazer/rails/cell.rb

```diff
@@ -7,13 +7,13 @@ def render(cell=nil, options={}, *, &block)
       render_cell(cell, options)
     end
 
-    def render_cell(cell, options)
-      options = options.reverse_merge(layout: true)
+    def render_cell(cell, context)
+      context = context.reverse_merge(layout: true)
```

Comment:
This `options` has nothing to do with the operation's `ctx`, please make it `options`. :heart: